sdk/schemaregistry/schema-registry-avro/src/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export * from "./schemaRegistryAvroSerializer";
```

Comment:
optional nit: I personally prefer manually exporting specific types to avoid accidentally exporting things, but I don't feel strongly about this, since we have the api-extractor output to catch it.